.changeset/new-doors-flow.md

```diff
@@ -0,0 +1,7 @@
+---
+"ledger-live-desktop": patch
```

Comment:
@sarneijim I think you want to be a minor here.

Also please split into three different changesets (yes it's a pain in the ass due to the release flow right now but hopefully we can work that out soon)